pypeit/scripts/ql.py

```diff
@@ -714,36 +714,40 @@ def get_parser(cls, width=None):
                             help='If standard star observations are automatically detected, '
                                  'ignore those frames.  Otherwise, they are included with the '
                                  'reduction of the science frames.')
-        parser.add_argument('--skip_display', dest='show', default=True, action='store_false',
-                            help='Run the quicklook without displaying any results.')
+        parser.add_argument('--skip_display', dest='skip_display', default=False, action='store_true',
```

Comment:
If the argument name is the same as the flag, you don't need the `dest` keyword.